wsl.md

```diff
@@ -0,0 +1,87 @@
+
+- WSL 2 is installed on Windows 11 using the default Linux distribution (Ubuntu):
+  - [Follow Microsoft's guide for installing WSL](https://learn.microsoft.com/en-us/windows/wsl/install)
+- Your WSL environment is setup to use a supported LTS version of Node.js and Git:
+  - [Follow Microsoft's guide for using Node.js in WSL](https://learn.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-wsl). If you install a Node version manager as they recommend, be sure to setup an **.nvmrc** in your project or set a default version to use (e.g., `nvm alias default 18`).
+  - [Follow Microsoft's guide for using Git in WSL](https://learn.microsoft.com/en-us/windows/wsl/tutorials/wsl-git)
+- Visual Studio Code installed in Windows with the [WSL Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl):
+  - [Follow Microsoft's guide for using Visual Studio Code with WSL](https://code.visualstudio.com/docs/remote/wsl)
+- [Windows Terminal](https://apps.microsoft.com/detail/9N0DX20HK701?hl=en-US&gl=US) is installed
+
+## Important: Your Expo project must be entirely on the Linux filesystem
+Your Windows environment and your WSL environments each have their own filesystems, and it is possible to interact with files on the Windows side from the WSL terminal prompt. However, once you are using WSL, your Expo project needs to exist on the Linux filesystem, and any source control operations need to be performed via the Linux version of git.
+
+If you're creating a new Expo project, open up Windows Terminal, open up a WSL tab, and then run `npx create-expo-app`. If you're working with an existing Expo project committed to a git repo, run `git clone` from a WSL terminal tab.
+
+If you use Visual Studio Code with the WSL extension and do all your git interactions via Visual Studio Code's git integration, Visual Studio Code will detect that the project files are on the Linux filesystem, and will open the appropriate terminal and use the Linux git installation. While Visual Studio Code is a Windows app, in this scenario, it is effectively remoting into the WSL Linux environment to perform these operations.
+
+### Warning signs that you're probably mixing Windows and Linux:
+- You run `pwd` and you see paths that start with `/mnt`.
+- You are in a WSL prompt interacting with a project that you originally cloned with Github Desktop for Windows.
+
+## Running your Expo project in WSL with no additional config
+If you have completed the setup above, you can open up a WSL terminal, navigate to your project folder, restore your dependencies, run `npx expo start`, and the Expo CLI will start up it would anywhere else. You can even press `w` to open up the web version of your app, and Chrome for Windows will open up, pointing to `http://localhost:8081`, and it will serve your app.
```

Comment:
```suggestion
If you have completed the setup above, you can open up a WSL terminal, navigate to your project folder, restore your dependencies, run `npx expo start`, and the Expo CLI will start up it would anywhere else. You can even press `w` to open up the web version of your app, and Chrome for Windows will open up, pointing to `http://localhost:8081`, and it will serve your app.

If you have completed the setup above, then:

- Open a WSL terminal.
- Navigate to your project folder and restore your dependencies.
- Run `npx expo start` to launch the Expo CLI.
- Press `w` to open the web version of your app. The web browser will launch and display your app at `http://localhost:8081`.
```

Let's break it down in steps. Also, what does "restore your dependencies" mean here? Do I need to install dependencies in my project?

Reply:
I don't actually want them to do this; I just want to describe what actually should work at this point. Maybe I can do that in fewer words, while also suggesting you need to keep going to enable developing with a device.

Reply:
Makes sense. To me, as a reader of this guide, it sounded like those were steps I need to take at this point while going through this and that's why I suggested them as steps.

Reply:
Just tried something more streamlined here

Reply:
Looks good!